test/sequential/test-watch-mode.mjs

```diff
@@ -104,14 +104,6 @@ describe('watch mode', { concurrency: true, timeout: 60_0000 }, () => {
     });
   });
 
-  it('should not watch when running an non-existing file', async () => {
-    const file = fixtures.path('watch-mode/non-existing.js');
-    const { stderr, stdout } = await spawnWithRestarts({ file, restarts: 0 });
-
-    assert.match(stderr, /code: 'MODULE_NOT_FOUND'/);
-    assert.strictEqual(stdout, `Failed running ${inspect(file)}\n`);
-  });
-
```

Comment:
maybe just tweak this test instead so that it asserts instead: `should watch when running an non-existing file`

Reply:
yeah I'l also  add a test for a missing dependency